packages/venia-concept/src/components/ProductFullDetail/productFullDetail.css

```diff
@@ -163,3 +165,14 @@
 .relatedTitle {
     composes: sectionTitle;
 }
+
+.stickyFooter {
+    position: sticky;
+    bottom: 0;
+}
+
+@media (min-width: 1024px) {
+    .stickyFooter {
+        display: none;
```

Comment:
Does it look incorrect at the larger sizes?

Reply:
See https://github.com/magento-research/pwa-studio/pull/659#discussion_r